### PR TITLE
MTP-2395 transport link endringer

### PR DESCRIPTION
**Pull request by petterkoding**
transport link styling
default
primary
primary--large
secondary
secondary--large

+

used as link list:
Lighter border color

### CODE REVIEW

File: mt-kit/core/css/src/global/_links.scss [hunk 1]
```diff
@@ -428,104 +428,112 @@ $transportlink-colors: 'error' !default;
...
 
     &:focus {
       border-color: var(--transportlink-background);
       box-shadow: $focus-box-shadow;
     }
-    &:hover {
-      background-color: transparent;
-    }
 
     &--primary {
```
Comment (lha047) [inline mt-kit/core/css/src/global/_links.scss:454]: For reuse I would create variables for default, hover and active and use in the &--primary and &-large. Same for the states in secondary. Maybe include variation and state in the names

File: mt-kit/core/css/src/global/_links.scss [hunk 2]
```diff
@@ -428,104 +428,112 @@ $transportlink-colors: 'error' !default;
...
+      }
+      &:active {
+        --transportlink-background: var(--color-mt-lys-furu);
+        @media (prefers-reduced-motion: no-preference) {
+          transition: background-color $transition-ms-300 ease;
+        }
+      }
 
       &-large {
-        @extend .no-underline;
```
Comment (lha047) [inline mt-kit/core/css/src/global/_links.scss:516]: Missing border radius
Screenshot 2024-08-01 at 12 11 40

File: mt-kit/core/css/src/global/_links.scss [hunk 3]
```diff
@@ -428,104 +428,112 @@ $transportlink-colors: 'error' !default;
     }
   }
   &.link--transport {
-    --transportlink-background: var(--color-#{$transportlink-default-color});
```
Comment (lha047) [inline mt-kit/core/css/src/global/_links.scss:431]: Missing underline and state styling
Screenshot 2024-08-01 at 12 11 48

**Review by lha047 — CHANGES_REQUESTED** [review]
See comments

File: mt-kit/core/css/src/global/_links.scss [hunk 4]
```diff
@@ -428,104 +428,112 @@ $transportlink-colors: 'error' !default;
...
+      --transportlink-background: var(--color-mt-green-300);
       padding: var(--transportlink-padding);
-      border-radius: $border-radius;
-
-      &:after {
-        background-image: var(--icon-arrow-primary);
-      }
 
       &:hover {
-        color: var(--mt-button-primary-color-hover);
```
Comment (lha047) [inline mt-kit/core/css/src/global/_links.scss:474]: Why not use the mixins?

File: mt-kit/core/css/src/global/_links.scss (hunk 3 above)
Comment (petterkoding) [reply mt-kit/core/css/src/global/_links.scss:431]: the default and `--primary` and `--secondary` (not large versions) will inherit text-decoration from the `.mt-link` class. Adjusting it thereafter by removing on hover (default) and setting to `none` on --primary and --secondary.

**Review by lha047 — CHANGES_REQUESTED** [review]
Screenshot 2024-08-05 at 13 53 27
Looks like you need som gaps after you removed margin on the psudo

File: mt-kit/core/css/src/utilities/colors.js [hunk 5]
```diff
@@ -135,7 +135,7 @@ module.exports = [
   },
   {
     name: 'mt-lys-furu',
-    color: '#8AC2AE',
+    color: '#9BCAB9',
```
Comment (petterkoding) [inline mt-kit/core/css/src/utilities/colors.js:138]: make global vars for new link colors

**Review by lha047 — APPROVED** [review]
👍

**Pull request merged**